scraper.go

```diff
@@ -153,8 +156,7 @@ func NewScraper(hostname, statefile, metalogfile string) *Scraper {
 	s := &Scraper{}
 	if hostname != "" {
 		s.Hostname = hostname
-	} else {
-		s.Hostname, _ = os.Hostname()
+		s.PCname,_ = os.Hostname()
```

Comment:
if i understand this correctly, if hostname is empty we will not be seeing the host or the PC name in the loggly message, please revisit this and add the else back to add the pcsname if hostname if empty.